tests/zfs-tests/tests/functional/raidz/raidz_002_pos.ksh

```diff
@@ -36,6 +36,6 @@
 #	is set to 5min, but actual runtime might be longer.
 #
 
-log_must $RAIDZ_TEST -S -t 300
+log_must $RAIDZ_TEST -S -t 40
```

Comment:
This test processes 'work-units' which are getting progressively bigger with time. So, when run on a fast CPU, with lot of cores, wall time discrepancy can get worse as time passes. See #5190 for fix.